content/documentation/v1.32/faq/general/how-do-i-access-kiai.adoc

```diff
@@ -5,7 +5,7 @@ weight: 10
 :icons: font
 
 *_This assumes that you have used the
-link:{{< ref "/documentation/v1.32/installation-guide" >}}[Installation guide] to
+link:{{< ref "../../installation-guide" >}}[Installation guide] to
```

Comment:
does this belong to the slack channel fix?

Reply:
If all else if fine, please approve and will merge.